src/js/yunohost/filters.js

```diff
@@ -14,6 +14,7 @@
         req.params.domains = [];
         req.api('GET', '/domains', {}, function(data) {
             req.params.domains = data.domains;
+            req.params.domains_default = data.default;
```

Comment:
According to https://github.com/YunoHost/yunohost/pull/1022#discussion_r493867909

```suggestion
            req.params.domains_default = data.main;
```